package.json

```diff
@@ -66,7 +66,6 @@
   "dependencies": {
     "array-differ": "^1.0.0",
     "array-uniq": "^1.0.2",
-    "babel-runtime": "^5.8.29",
```

Comment:
No, still needed. It's for stuff that can't be statically transformed, like async functions.